app/src/main/java/com/codesoom/assignment/application/ProductQueryService.java

```diff
@@ -0,0 +1,44 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.exception.ProductNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 상품 조회 담당
+ */
+@Service
+public class ProductQueryService {
+
+    private final ProductRepository productRepository;
+
+    public ProductQueryService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 상품 전체 목록 반환
+     */
+    public List<Product> getProducts() {
+        Iterable<Product> source = productRepository.findAll();
+        List<Product> products = new ArrayList<>();
+        source.forEach(products::add);
+        return products;
+    }
+
+    /**
+     * 단일 상품 반환
+     *
+     * @param productId 상품 아이디
+     * @throws ProductNotFoundException 상품을 찾을 수 없을 때 던집니다.
+     */
```

Comment:
메인 설명은 마침표로 끝나는 문장형으로 작성해 주세요. (JavaDoc의 메인 설명은 외부에 공개하는 문장이라고 생각해 보세요.)
`@throws`는 문장형이 아니라 어떤 경우에 대한 것인지 설명해 주는 정도로 충분합니다.
```suggestion
    /**
     * 단일 상품을 반환합니다.
     *
     * @param productId 상품 아이디
     * @throws ProductNotFoundException 상품을 찾을 수 없을 경우
     */
```

Reply:
훨씬 자연스러운 거 같습니다.